lang/java/pom.xml

```diff
@@ -35,6 +35,8 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jdkVersion>1.6</jdkVersion>
+    <jdk8Version>1.8</jdk8Version> <!-- Some tests need java 8 or newer -->
```

Comment:
I don't like the naming of this properly. I would rather name it after its purpose e.g. testJdkVersion.